tests/resource/test_error.py

```diff
@@ -81,3 +80,27 @@ def test_get_error(self):
 
         res = SimpleError.get_error(approx_op, exact_op)
         assert res == 0.5
+
+
+class TestErrorOperation:  # pylint: disable=too-few-public-methods
+    """Test the abstract error operation class."""
+
+    def test_error_method(self):
+        """Test that error method works as expected"""
+
+        class SimpleErrorOperation(ErrorOperation):  # pylint: disable=too-few-public-methods
+            @property
+            def error(self):
+                return len(self.wires)
+
+        no_error_op = SimpleErrorOperation(wires=[1, 2, 3])
+        assert no_error_op.error == 3
+
+    def test_no_error_method(self):
+        """Test error is raised if the method is not defined."""
+        with pytest.raises(TypeError, match="Can't instantiate abstract class"):
```

Comment:
Is it supposed to be abstract?

Reply:
Yes, it is required to be implemented otherwise the class doesn't make sense